src/Resources/contao/library/Contao/Environment.php

```diff
@@ -332,7 +292,7 @@ protected static function httpXForwardedHost()
 	 */
 	protected static function ssl()
```

Comment:
I know you did follow exactly the 1:1 mapping here but I think we could safely rely on SF's `isSecure()` here.